sdk/storage/storage-blob/BreakingChanges.md

```diff
@@ -25,7 +25,7 @@
 - Updates to `BlockBlobClient.uploadStream`
   - `maxBuffers` attribute of is renamed to `maxConcurrency`
 - Bug Fix - The page object returned from `ContainerClient.listContainers` had its `containerItems` property set to an empty string instead of an empty array if the storage account has no blob containers. The issue is fixed in this new release.
-- The default browser bundle has been removed from the npm package. Bundling your application with a bundler such as Webpack is the recommended approach to building a browser bundle. For details on how to do this, please refer to our [bundling documentation](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md).
+- The default browser bundle has been removed from the npm package. Bundling your application with a bundler such as Webpack is the recommended approach to building a browser bundle. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
```

Comment:
@bterlson did you create this aka link? I'm only wondering as we seem to be accomplishing these in various places across all our repo's and I'm not sure how we are ever going to manage them all. I'm not sure if there is an easy way to assign ownership to a group for example.

Also it looks like the link is directly going to master right now is that what we want? Should it be versioned so it points to docs at the point this is released or will the latest docs always be applicable?

Reply:
I did create it, and set management to the adpteam SG, which should allow any of us to manage it.

The latest docs should always be applicable. We will definitely be iterating on these docs over the next few weeks so I definitely want it to reflect updates for now.